src/Components/AnimalList.js

```diff
@@ -1,21 +0,0 @@
-import React, {Component} from 'react'
-import axios from 'axios'
-
-const AnimalList = (props) => {
-    return (
-        <div>
-            <div className='title'>
-            <h2>Animals</h2>
-            </div>
-            {props.animals.map(animal =>(//add key?
-            <div>    
```

Comment:
to add the key, you'd just give this outer div a key prop, like 
```
<div key={animal.id}>
```
however - since you weren't posting incrementing animal ids, this may have broken down, but it should work once you get the post fixed.